app/datatables/registration_datatable.rb

```diff
@@ -41,7 +41,7 @@ def data
         email:                    record.email,
         accepted_code_of_conduct: !!record.accepted_code_of_conduct, # rubocop:disable Style/DoubleNegation
         edit_url:                 edit_admin_conference_registration_path(conference, record),
-        DT_RowId:                 record.id
+        DT_RowId:                 "#{record.class.name.parameterize}-#{record.id}"
```

Comment:
[dom_id](https://api.rubyonrails.org/classes/ActionView/RecordIdentifier.html#method-i-dom_id) does that for you

Reply:
Perfect, thanks.